index.js

```diff
@@ -57,12 +57,6 @@ function validate(word) {
 }
 
 function stonksify(word) {
-  let modifiedCharacters = Array.apply(null, Array(word.length)).map(
-    function() {
-      return 0;
-    }
-  );
-
```

Comment:
Oh i see we're not using this anymore. Disregard other comments then that don't apply.